packages/react-core/src/components/LoginPage/LoginPage.tsx

```diff
@@ -13,33 +13,35 @@ import { LoginMainBody } from './LoginMainBody';
 import { LoginMainFooter } from './LoginMainFooter';
 
 export interface LoginPageProps extends React.HTMLProps<HTMLDivElement> {
-  /** Anything that can be rendered inside of the LoginPage (e.g. <LoginPageForm>) */
+  /** Anything that can be rendered inside of the login page (e.g. <LoginPageForm>) */
   children?: React.ReactNode;
-  /** Additional classes added to the LoginPage. */
+  /** Additional classes added to the login page */
   className?: string;
-  /** Attribute that specifies the URL of the brand image for the LoginPage */
+  /** Attribute that specifies the URL of the brand image for the login page */
   brandImgSrc?: string;
-  /** Attribute that specifies the alt text of the brand image for the LoginPage. */
+  /** Attribute that specifies the alt text of the brand image for the login page */
   brandImgAlt?: string;
-  /** Attribute that specifies the URL of the background image for the LoginPage */
+  /** Attribute that specifies the URL of the background image for the login page */
   backgroundImgSrc?: string | BackgroundImageSrcMap;
-  /** Attribute that specifies the alt text of the background image for the LoginPage. */
+  /** Attribute that specifies the alt text of the background image for the login page */
   backgroundImgAlt?: string;
-  /** Content rendered inside of the Text Component of the LoginPage */
+  /** Content rendered inside of the text component of the login page */
   textContent?: string;
-  /** Items rendered inside of the Footer List Component of the LoginPage */
+  /** Items rendered inside of the footer list component of the login page */
   footerListItems?: React.ReactNode;
-  /** Adds list variant styles for the Footer List component of the LoginPage. The only current value is'inline' */
+  /** Adds list variant styles for the footer list component of the login page. The only current value is'inline' */
   footerListVariants?: ListVariant.inline;
-  /** Title for the Login Main Body Header of the LoginPage */
+  /** Title for the login main body header of the login page */
   loginTitle: string;
-  /** Subtitle for the Login Main Body Header of the LoginPage */
+  /** Subtitle for the login main body header of the login page */
   loginSubtitle?: string;
-  /** Content rendered inside of Login Main Footer Band to display a sign up for account message */
+  /** Header utilities for the login main body header of the login page */
+  headerUtilities?: React.ReactNode;
```

Comment:
could we label this one beta? Then we can merge.

Reply:
@tlabaj how do I label something as beta?

Reply:
Hi @andyyvo You just need to but the `@beta` to the  beginning of the prop description.  This give us the flexibility to improve on API (including breaking change) if need be.  Once we decide feature is stable, we promote it out of beta and remove the tag.
`/** @beta Header utilities for the login main body header of the login page */`

Reply:
@tlabaj would you also like to see this on the LoginMainHeader.tsx prop as well?